CVLByExample/TransientStorage/Hooks/MockMutexer.sol

```diff
@@ -15,17 +15,29 @@ contract MockMutexer is Mutexer {
 
     constructor() contractLock{}
 
-    // getters
-    function lockValue() external returns (uint){
-        return uint(Mutex.Locked);
-    }
-    function getContractLock() public returns (uint256){
-        return CONTRACT_LOCK;
+    function isLocked() external returns (bool){
+        Mutex value;
+        uint256 key = CONTRACT_LOCK;
+        assembly {
+             value := tload(key)
+        }
+        return value == Mutex.Locked;
```

Comment:
This should actually be `_tload()` which is private in the parent contract and cannot be called here.